Python/ceval.c

```diff
@@ -4812,7 +4812,7 @@ _PyEval_EvalFrameDefault(PyThreadState *tstate, _PyInterpreterFrame *frame, int
             function = PEEK(total_args + 1);
             int positional_args = total_args - KWNAMES_LEN();
             // Check if the call can be inlined or not
-            if (Py_TYPE(function) == &PyFunction_Type && tstate->interp->eval_frame == NULL) {
+            if (0 && Py_TYPE(function) == &PyFunction_Type && tstate->interp->eval_frame == NULL) {
```

Comment:
This will need to be controlled likely by configuration so is only active in debugging.

Reply:
If you use `py_trampoline_evaluator`, then there is no need for the additional test and this can be enabled all the time

Reply:
Done!